<!-- Launches an UVC camera, the ball detector and its visualization  -->
<launch>

        <node pkg="usb_cam" type="usb_cam_node" name="usb_cam_node" output="screen">
            <param name="video_device" type="string" value="/dev/video0" />
            <param name="image_width" type="int" value="1280" />
            <param name="image_height" type="int" value="720" />
            <param name="framerate " type="int" value="27" />
            <param name="auto_exposure" type="bool" value="False" />
            <param name="camera_frame_id" type="string" value="cam_link" />
            <param name="camera_name" type="string" value="camera" />
            <param name="auto_white_balance" type="bool" value="False" />
            <param name="autofocus" type="bool" value="False" />
		</node>

</launch>
